seirmo/__init__.py

```diff
@@ -12,3 +12,7 @@
 from .simulation import (
     SimulationController
 )
+
+from .slider import (
+    _SliderComponent
```

Comment:
see above, you would basically import the apps module here, and apps is a directory with the _sliders.py. (it makes sense to keep the sliders.py private, because we want nobody to use this file directly, and that can be signalled by preprending `_`.